features/guidance/suffix-changes.feature

```diff
@@ -3,7 +3,7 @@ Feature: Suppress New Names on dedicated Suffices
 
     Background:
         Given the profile "car"
-        Given a grid size of 10 meters
+        Given a grid size of 2000 meters
```

Comment:
Why do we need this change here? Because the suffix change only triggers at a specific distance?